R/qenv-constructor.R

```diff
@@ -10,13 +10,11 @@
 #' `qenv()` instantiates a `qenv` with an empty environment.
 #' Any changes must be made by evaluating code in it with `eval_code` or `within`, thereby ensuring reproducibility.
 #'
-#' `new_qenv()` (`r badge("deprecated")` and not recommended)
-#' can instantiate a `qenv` object with data in the environment and code registered.
-#'
 #' @name qenv
 #'
-#' @return `qenv` and `new_qenv` return a `qenv` object.
+#' @return `qenv` returns a `qenv` object.
```

Comment:
```suggestion
#' @return `qenv()` returns a `qenv` object.
```

Or simplify to "@return `qenv` object" as there's gonna be just `qenv` documented on this site

Reply:
Took the second approach